src/lang/en.json

```diff
@@ -79,6 +79,7 @@
     "customerSupport.customerSupport": "Customer Support",
     "downloadFile.download": "Download File",
     "downloadFile.fileId": "File ID",
+    "downloadFile.invalidFileId": "File \"{0}\" Does Not Exist",
```

Comment:
> File does not exist.

or

> File 0.0.210 does not exist.

Reply:
should we also display the hedera  "invalid file id" error as an input error, or leave that as an alert?